services/mlmodel/tflite_cpu/tflite_cpu.go

```diff
@@ -0,0 +1,241 @@
+// Package tflitecpu is meant to implement a tfliteCPU in the ML model service
+package tflitecpu
+
+import (
+	"context"
+	fp "path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"go.opencensus.io/trace"
+
+	"go.viam.com/rdk/config"
+	inf "go.viam.com/rdk/ml/inference"
+	"go.viam.com/rdk/ml/inference/tflite_metadata"
+	"go.viam.com/rdk/registry"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/robot"
+	"go.viam.com/rdk/services/mlmodel"
+	"go.viam.com/rdk/services/vision"
+	"go.viam.com/rdk/utils"
+)
+
+func init() {
+	registry.RegisterService(vision.Subtype, resource.DefaultServiceModel, registry.Service{
+		RobotConstructor: func(ctx context.Context, r robot.Robot, config config.Service, logger golog.Logger) (interface{}, error) {
```

Comment:
Don't use a robot constructor if you can help it, instead use a normal constructor and use dependencies. You can see how it's done in SLAM.  

https://github.com/viamrobotics/rdk/blob/main/services/slam/builtin/builtin.go#L74